src/internal/buffers.js

```diff
@@ -29,6 +36,11 @@ function ringBuffer(limit = 10, overflowAction) {
             pushIndex = (pushIndex + 1) % limit
             popIndex = pushIndex
             break
+          case ON_OVERFLOW_ACCOMODATE:
+            doubledLimit = 2 * limit
+            arr.length = length = limit = doubledLimit
```

Comment:
length shouldn't be modified because it denotes the number of items in the array not he length of the array